test/byContract/IexecPoco/IexecPoco1.test.ts

```diff
@@ -206,9 +265,93 @@ describe('IexecPoco1', () => {
             });
         });
     });
-    // TODO
-    describe('Verify presignature', () => {});
-    describe('Verify presignature or signature', () => {});
+
+    describe('Verify presignature', () => {
+        ['app', 'dataset', 'workerpool', 'requester'].forEach((asset) => {
+            ['verifyPresignature', 'verifyPresignatureOrSignature'].forEach(
+                (verifyPreSignatureFunction) => {
+                    beforeEach(() => {
+                        providerAddress = orderManagement[asset].providerAddress;
+                        orderHash = orderManagement[asset].orderHash;
+                        iexecPocoManageOrderTx = orderManagement[asset].iexecPocoManageOrderTx;
+                    });
+
+                    it(`Should ${verifyPreSignatureFunction} when the presignature is valid for ${asset}`, async () => {
+                        await iexecPocoManageOrderTx().then((tx) => tx.wait());
```

Comment:
```suggestion
                        await iexecPocoManageOrderWithSignOperation(order).connect(providerAddress).then((tx) => tx.wait());
```
(= I think your iexecPocoManageOrderWithSignOperation could be generic and declared one once on top instead of declaring it multiple times in orderManagement)